src/python/py/models/builder.py

```diff
@@ -853,14 +853,26 @@ def make_matmul_lora(self, matmul, basename, root_input, **kwargs):
         matmul_A_name = self.make_matmul_op(matmul.lora_A.default, matmul_A_basename, root_input=root_input)
         lora_A = f"{matmul_A_name}/output_0"
 
-        matmul.lora_B.default.weight *= matmul.scaling["default"]
+        from peft import PeftConfig
+        peft_config = PeftConfig.from_pretrained(extra_options["adapter_path"], trust_remote_code=True)
```

Comment:
Can we create a `lora` section in `self.matmul_attrs` and add the attributes that we need there? The default values can be 0 when LoRA isn't used.

```python
self.matmul_attrs = {
    "use_lora": is_lora,
    "lora": {
        "lora_alpha": config.lora_alpha if is_lora else 0,
        "r": config.r if is_lora else 0
    }
}
```